docs/development/contribution/how_to_contribute_code.md

```diff
@@ -79,10 +78,10 @@ To skip test
 ./mvnw install -DskipTests
 ```
 
-To build with specific spark / hadoop version
+To build with specific spark version
 
 ```bash
-./mvnw install -Dspark.version=x.x.x -Dhadoop.version=x.x.x
+./mvnw install -Dspark.version=x.x.x
```

Comment:
the `hadoop.version` still works, for example, users may want to use hadoop 3.2.2 client to match the server-side version.

Reply:
Adjusted.